### PR TITLE
torch frontend tensor functions

### PR DESCRIPTION
**Pull request by AnnaTz**

### CODE REVIEW

**Comment by AnnaTz** [comment]
Close #4975

**Comment by AnnaTz** [comment]
Close #4977

File: ivy/functional/frontends/torch/tensor_functions.py [hunk 1]
```diff
@@ -2,10 +2,6 @@
 import ivy
```
Comment (simonetgordon) [inline ivy/functional/frontends/torch/tensor_functions.py:4]: Is this supposed to be deleted?

**Comment by simonetgordon** [comment]
Hi @AnnaTz, `is_non_zero` is passing - nice! But `numel` and `is_floating_point` are failing. Check Details -> Run Frontend Tests then search for these functions in the error output. You will be able to see the error stacktraces. You should be able to reproduce the errors by testing locally. Let me know if you need any help!

File: ivy/functional/frontends/torch/tensor_functions.py (hunk 1 above)
Comment (AnnaTz) [reply ivy/functional/frontends/torch/tensor_functions.py]: Oops, I'm surprised I didn't notice that. Thanks, I recovered it.

**Comment by AnnaTz** [comment]
> Hi @AnnaTz, `is_non_zero` is passing - nice! But `numel` and `is_floating_point` are failing. Check Details -> Run Frontend Tests then search for these functions in the error output. You will be able to see the error stacktraces. You should be able to reproduce the errors by testing locally. Let me know if you need any help!

It's strange, but sometimes tests pass locally and yet fail in the workflows.

**Review by simonetgordon — APPROVED** [review]
lgtm!

**Pull request merged**